presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlDistributedQueries.java

```diff
@@ -85,5 +81,16 @@ protected TestTable createTableWithDefaultColumns()
                         "col_required2 BIGINT NOT NULL)");
     }
 
+    @Test
+    @Override
+    public void testDelete()
+    {
+        // PostgreSql connector currently does not support row-by-row delete
```

Comment:
I am not getting this. If row-by-row is not supported for any of the connectors, then i do not understand why `io.prestosql.plugin.jdbc.QueryBuilder#buildDeleteSql` does tuple domain -> WHERE conversion

Reply:
tuple domain is not used to delete row-by-row, by rather to remove by predicate. By row-by-row I mean to delete query execution in Trino where we collect ids of rows to be deleted and then we remove them (`beginDelete` and `finishDelete` methods).